Assignment2/Q1/src/BinaryTree.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Created by Alice Colt.
+ *
+ * Implementaiton of a Binary Tree with functions for insert, serach and takeAncestors
```

Comment:
Nit: Two typos in the comment at the top of the file give a bad impression. Most editors can be set up to spell-check comments and it's helpful for giving a good impression.

(Discussing typos in comments can seem very pedantic, but typo-based bugs in code are frustratingly common, so being wary of typos is helpful.)